components/camel-cassandraql/src/main/java/org/apache/camel/component/cassandra/CassandraEndpoint.java

```diff
@@ -165,7 +169,8 @@ protected CqlSessionBuilder createSessionBuilder() {
 
         if (extraTypeCodecs != null) {
             String[] c = extraTypeCodecs.split(",");
-            System.err.println(c.toString());
+            LOG.error(Arrays.toString(c));
```

Comment:
We should not log this at error level, instead at debug or trace level

Reply:
Fixed, thanks!